Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "237595339.61877787" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "1022234173.6669613" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 389, in writeModel
    _format_83(coords[1]), _format_83(coords[2]), symbol)
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "1505499294.9672472" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "114336023.61105627" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-124205679.60838473" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "188604376.4607532" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-137539183.4522279" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-770864704.2687693" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-719380893.407213" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-427039855.2830095" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "246882347.00257516" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-1087477792.326562" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "181169072.88608745" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-350944053.36454237" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "500698236.0669759" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "121547165.55039126" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "252630235.65485325" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-295192572.3731912" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-274560031.5125451" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-334841446.707117" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-121873461.18610513" could not be represented in a width-8 field
./run_md.py:723: UserWarning: No output for .//run_0. MD run failed with The periodic box size has decreased to less than twice the nonbonded cutoff..
  warnings.warn(f"No output for {output_dir_replicate}. MD run failed with {result}.")
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 389, in writeModel
    _format_83(coords[1]), _format_83(coords[2]), symbol)
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "739995700.1665356" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-608471201.0798118" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-114305476.45083028" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 389, in writeModel
    _format_83(coords[1]), _format_83(coords[2]), symbol)
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-74902185.12749864" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "994306284.2663473" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "594692824.02063" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-194752236.79122448" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 389, in writeModel
    _format_83(coords[1]), _format_83(coords[2]), symbol)
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "236476160.31084222" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
./run_md.py:723: UserWarning: No output for .//run_0. MD run failed with The periodic box size has decreased to less than twice the nonbonded cutoff..
  warnings.warn(f"No output for {output_dir_replicate}. MD run failed with {result}.")
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-140486828.138562" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "532444229.78537345" could not be represented in a width-8 field
2022053_parsley_1.0.0.xml
2100437_parsley_1.2.0.xml
1100989_parsley_1.0.0.xml
7154682_sage_2.0.0.xml
4116140_parsley_1.1.0.xml
2014075_parsley_1.0.0.xml
4020664_parsley_1.2.0.xml
4506226_parsley_1.0.0.xml
1519191_parsley_1.3.0.xml
2220312_sage_2.0.0.xml
2105612_parsley_1.3.0.xml
2208405_sage_2.0.0.xml
2238806_parsley_1.2.0.xml
4034456_parsley_1.0.0.xml
7236728_sage_2.0.0.xml
2203255_sage_2.0.0.xml
1100992_parsley_1.1.0.xml
1513282_sage_2.0.0.xml
7241270_sage_2.0.0.xml
2107938_sage_2.0.0.xml
2202092_parsley_1.1.0.xml
4507232_sage_2.0.0.xml
2103490_parsley_1.0.0.xml
4501704_sage_2.0.0.xml
1513289_sage_2.0.0.xml
7121743_parsley_1.1.0.xml
2105612_parsley_1.1.0.xml
2100349_parsley_1.0.0.xml
7151616_parsley_1.2.0.xml
7110463_parsley_1.2.0.xml
2237525_parsley_1.0.0.xml
2004882_parsley_1.1.0.xml
2203255_parsley_1.3.0.xml
2220312_parsley_1.2.0.xml
4035731_parsley_1.2.0.xml
1504357_parsley_1.1.0.xml
4501704_parsley_1.1.0.xml
2205937_sage_2.0.0.xml
2203540_parsley_1.2.0.xml
1519191_parsley_1.0.0.xml
2241317_sage_2.0.0.xml
2105612_parsley_1.2.0.xml
7221763_sage_2.0.0.xml
4508226_parsley_1.2.0.xml
2218498_parsley_1.2.0.xml
2238379_parsley_1.2.0.xml
2231307_parsley_1.3.0.xml
2221170_sage_2.0.0.xml
2223130_parsley_1.0.0.xml
7238218_sage_2.0.0.xml
2234322_parsley_1.2.0.xml
1511801_sage_2.0.0.xml
7045653_parsley_1.2.0.xml
2007511_sage_2.0.0.xml
2022053_sage_2.0.0.xml
1511801_parsley_1.2.0.xml
2022073_parsley_1.0.0.xml
4515127_parsley_1.2.0.xml
2210848_parsley_1.3.0.xml
2221170_parsley_1.3.0.xml
4025034_parsley_1.0.0.xml
2203540_sage_2.0.0.xml
2229006_parsley_1.3.0.xml
7100390_sage_2.0.0.xml
7206297_parsley_1.1.0.xml
2022073_parsley_1.3.0.xml
7219800_parsley_1.2.0.xml
4507100_parsley_1.3.0.xml
7216950_parsley_1.3.0.xml
7238218_parsley_1.1.0.xml
4116819_parsley_1.0.0.xml
2002929_parsley_1.1.0.xml
2007371_parsley_1.2.0.xml
7235707_parsley_1.3.0.xml
2236470_parsley_1.2.0.xml
2003630_parsley_1.1.0.xml
2206832_parsley_1.2.0.xml
2238379_parsley_1.3.0.xml
4116203_parsley_1.1.0.xml
2006075_parsley_1.3.0.xml
7110463_parsley_1.0.0.xml
4516576_parsley_1.3.0.xml
2007371_sage_2.0.0.xml
2020594_parsley_1.1.0.xml
2100840_sage_2.0.0.xml
1519825_parsley_1.0.0.xml
4020664_parsley_1.3.0.xml
2219974_sage_2.0.0.xml
1519868_sage_2.0.0.xml
4501704_parsley_1.2.0.xml
2234037_sage_2.0.0.xml
2220312_parsley_1.0.0.xml
2206832_sage_2.0.0.xml
2015070_parsley_1.3.0.xml
2018826_parsley_1.1.0.xml
4037918_parsley_1.2.0.xml
2002966_parsley_1.0.0.xml
2223130_sage_2.0.0.xml
2100350_parsley_1.1.0.xml
7121743_parsley_1.3.0.xml
7219235_parsley_1.2.0.xml
2007511_parsley_1.3.0.xml
2220312_parsley_1.1.0.xml
4027137_parsley_1.0.0.xml
7213183_parsley_1.3.0.xml
4114396_sage_2.0.0.xml
2022053_parsley_1.1.0.xml
7242545_parsley_1.0.0.xml
2208843_sage_2.0.0.xml
2215272_sage_2.0.0.xml
2017090_parsley_1.1.0.xml
2020594_sage_2.0.0.xml
2216200_parsley_1.1.0.xml
2104943_parsley_1.0.0.xml
2008450_parsley_1.0.0.xml
1100989_parsley_1.1.0.xml
2100662_parsley_1.0.0.xml
2017090_parsley_1.3.0.xml
2236004_parsley_1.3.0.xml
2105612_parsley_1.0.0.xml
2020594_parsley_1.2.0.xml
2231730_parsley_1.3.0.xml
7200790_parsley_1.2.0.xml
3500018_parsley_1.3.0.xml
3500018_parsley_1.0.0.xml
2100840_parsley_1.0.0.xml
4037896_parsley_1.3.0.xml
7118053_sage_2.0.0.xml
4027137_parsley_1.1.0.xml
2203632_parsley_1.1.0.xml
4035731_sage_2.0.0.xml
7219429_parsley_1.3.0.xml
1100992_parsley_1.3.0.xml
2238479_parsley_1.1.0.xml
1516412_parsley_1.3.0.xml
7052009_sage_2.0.0.xml
2203632_parsley_1.0.0.xml
2217222_parsley_1.0.0.xml
2242758_parsley_1.3.0.xml
4038405_parsley_1.2.0.xml
7214681_sage_2.0.0.xml
2020594_parsley_1.0.0.xml
4116819_parsley_1.3.0.xml
7232896_parsley_1.2.0.xml
2234322_sage_2.0.0.xml
7219429_parsley_1.1.0.xml
2100662_parsley_1.1.0.xml
2103490_sage_2.0.0.xml
2238806_sage_2.0.0.xml
2208497_parsley_1.2.0.xml
2005486_parsley_1.0.0.xml
2004431_parsley_1.3.0.xml
7052848_parsley_1.0.0.xml
7241270_parsley_1.1.0.xml
4501704_parsley_1.0.0.xml
7121743_parsley_1.2.0.xml
2238806_parsley_1.1.0.xml
2231730_parsley_1.0.0.xml
2212135_parsley_1.3.0.xml
7219429_parsley_1.2.0.xml
2311654_sage_2.0.0.xml
7238218_parsley_1.3.0.xml
2210911_parsley_1.2.0.xml
1516412_parsley_1.1.0.xml
7220218_sage_2.0.0.xml
2215272_parsley_1.3.0.xml
2223130_parsley_1.2.0.xml
4515127_parsley_1.3.0.xml
2242758_parsley_1.2.0.xml
2234037_parsley_1.1.0.xml
7121745_parsley_1.1.0.xml
7206516_parsley_1.3.0.xml
1556231_parsley_1.3.0.xml
4501703_parsley_1.2.0.xml
2206542_parsley_1.1.0.xml
2100147_parsley_1.2.0.xml
2012958_sage_2.0.0.xml
2238479_parsley_1.2.0.xml
7111201_parsley_1.1.0.xml
2200608_sage_2.0.0.xml
1511816_parsley_1.0.0.xml
2200608_parsley_1.0.0.xml
1511816_parsley_1.3.0.xml
1010060_parsley_1.0.0.xml
1519191_parsley_1.2.0.xml
2238379_parsley_1.1.0.xml
2203241_parsley_1.2.0.xml
2107937_parsley_1.0.0.xml
2200633_parsley_1.2.0.xml
2012828_parsley_1.3.0.xml
2229006_parsley_1.0.0.xml
4114396_parsley_1.2.0.xml
7219235_sage_2.0.0.xml
2210911_parsley_1.3.0.xml
2234322_parsley_1.0.0.xml
4507232_parsley_1.1.0.xml
2105702_sage_2.0.0.xml
2200608_parsley_1.3.0.xml
2004882_parsley_1.3.0.xml
2008677_parsley_1.2.0.xml
2224771_parsley_1.3.0.xml
2219974_parsley_1.0.0.xml
2242758_parsley_1.0.0.xml
2203540_parsley_1.3.0.xml
2241317_parsley_1.0.0.xml
1511801_parsley_1.1.0.xml
1556231_sage_2.0.0.xml
1010060_parsley_1.3.0.xml
4034456_parsley_1.2.0.xml
1513289_parsley_1.0.0.xml
7118053_parsley_1.3.0.xml
4114396_parsley_1.1.0.xml
2100603_parsley_1.3.0.xml
7151616_parsley_1.0.0.xml
2224771_sage_2.0.0.xml
2205643_sage_2.0.0.xml
2105702_parsley_1.2.0.xml
2007511_parsley_1.1.0.xml
7111201_parsley_1.2.0.xml
8100793_parsley_1.2.0.xml
2203255_parsley_1.1.0.xml
2105702_parsley_1.1.0.xml
7121745_sage_2.0.0.xml
1010060_parsley_1.2.0.xml
2205937_parsley_1.2.0.xml
2300006_sage_2.0.0.xml
2205643_parsley_1.1.0.xml
2008450_parsley_1.2.0.xml
2103490_parsley_1.1.0.xml
4037896_sage_2.0.0.xml
7220218_parsley_1.0.0.xml
2235036_sage_2.0.0.xml
2231503_parsley_1.0.0.xml
2311654_parsley_1.3.0.xml
2100348_parsley_1.1.0.xml
2207637_parsley_1.3.0.xml
2215272_parsley_1.0.0.xml
7100390_parsley_1.1.0.xml
7206934_parsley_1.3.0.xml
7118053_parsley_1.2.0.xml
2012958_parsley_1.0.0.xml
2007511_parsley_1.0.0.xml
2209888_sage_2.0.0.xml
1011023_parsley_1.1.0.xml
2018826_sage_2.0.0.xml
1511816_sage_2.0.0.xml
2234322_parsley_1.1.0.xml
2012828_sage_2.0.0.xml
4107711_parsley_1.2.0.xml
2206542_parsley_1.2.0.xml
1100992_sage_2.0.0.xml
2212078_parsley_1.3.0.xml
2203255_parsley_1.2.0.xml
2218498_parsley_1.0.0.xml
2201001_sage_2.0.0.xml
2100865_parsley_1.0.0.xml
2208843_parsley_1.2.0.xml
2219243_sage_2.0.0.xml
7045653_parsley_1.1.0.xml
2004431_parsley_1.1.0.xml
7059603_parsley_1.0.0.xml
4037916_parsley_1.3.0.xml
7219235_parsley_1.0.0.xml
4515127_sage_2.0.0.xml
4508226_sage_2.0.0.xml
7110463_parsley_1.1.0.xml
7236113_sage_2.0.0.xml
7121743_parsley_1.0.0.xml
7238221_sage_2.0.0.xml
2020594_parsley_1.3.0.xml
4037896_parsley_1.0.0.xml
1513592_sage_2.0.0.xml
2015769_parsley_1.3.0.xml
2006099_parsley_1.1.0.xml
2101393_parsley_1.2.0.xml
2003630_parsley_1.0.0.xml
2236470_parsley_1.0.0.xml
2202092_parsley_1.3.0.xml
7242545_sage_2.0.0.xml
4035731_parsley_1.3.0.xml
2240041_parsley_1.2.0.xml
7206297_sage_2.0.0.xml
2101393_parsley_1.1.0.xml
2238806_parsley_1.3.0.xml
1100992_parsley_1.0.0.xml
7235707_parsley_1.2.0.xml
7052009_parsley_1.0.0.xml
2007155_sage_2.0.0.xml
2226624_parsley_1.2.0.xml
2234322_parsley_1.3.0.xml
4116140_parsley_1.0.0.xml
8100793_parsley_1.3.0.xml
2220694_parsley_1.2.0.xml
1549860_parsley_1.0.0.xml
2203632_parsley_1.2.0.xml
2208843_parsley_1.3.0.xml
4516576_parsley_1.0.0.xml
1504385_parsley_1.3.0.xml
4116140_parsley_1.3.0.xml
7241270_parsley_1.2.0.xml
1513592_parsley_1.1.0.xml
2015070_parsley_1.1.0.xml
7238218_parsley_1.0.0.xml
1518526_parsley_1.1.0.xml
2242758_parsley_1.1.0.xml
2209888_parsley_1.0.0.xml
4034456_parsley_1.1.0.xml
2229458_parsley_1.3.0.xml
7052848_parsley_1.1.0.xml
7111201_parsley_1.0.0.xml
7210454_parsley_1.3.0.xml
7238221_parsley_1.3.0.xml
2240041_parsley_1.3.0.xml
2212078_parsley_1.2.0.xml
7206161_sage_2.0.0.xml
4037916_sage_2.0.0.xml
4107711_parsley_1.3.0.xml
1100989_parsley_1.2.0.xml
7100390_parsley_1.2.0.xml
2008677_parsley_1.1.0.xml
7236113_parsley_1.3.0.xml
2220694_sage_2.0.0.xml
4025034_parsley_1.1.0.xml
2212135_parsley_1.0.0.xmlTraceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-286416626.99537337" could not be represented in a width-8 field
./run_md.py:723: UserWarning: No output for .//run_0. MD run failed with The periodic box size has decreased to less than twice the nonbonded cutoff..
  warnings.warn(f"No output for {output_dir_replicate}. MD run failed with {result}.")
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-629984831.2001632" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "648189004.9590529" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "685291658.7783403" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-390546637.3239744" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "1454737448.5506067" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
./run_md.py:723: UserWarning: No output for .//run_0. MD run failed with The periodic box size has decreased to less than twice the nonbonded cutoff..
  warnings.warn(f"No output for {output_dir_replicate}. MD run failed with {result}.")
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "125093627.34922199" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-339779168.0826822" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-29832057.82036256" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-244910980.0835604" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "676412535.6750913" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 389, in writeModel
    _format_83(coords[1]), _format_83(coords[2]), symbol)
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-461180515.9008015" could not be represented in a width-8 field
./run_md.py:723: UserWarning: No output for .//run_0. MD run failed with The periodic box size has decreased to less than twice the nonbonded cutoff..
  warnings.warn(f"No output for {output_dir_replicate}. MD run failed with {result}.")
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-146813805.22154832" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "1973280899.046359" could not be represented in a width-8 field
./run_md.py:723: UserWarning: No output for .//run_0. MD run failed with The periodic box size has decreased to less than twice the nonbonded cutoff..
  warnings.warn(f"No output for {output_dir_replicate}. MD run failed with {result}.")
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "218956238.1735337" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "466792479.4321691" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 389, in writeModel
    _format_83(coords[1]), _format_83(coords[2]), symbol)
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "310248044.1787474" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 389, in writeModel
    _format_83(coords[1]), _format_83(coords[2]), symbol)
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "242914318.42499298" could not be represented in a width-8 field
./run_md.py:723: UserWarning: No output for .//run_0. MD run failed with The periodic box size has decreased to less than twice the nonbonded cutoff..
  warnings.warn(f"No output for {output_dir_replicate}. MD run failed with {result}.")
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 389, in writeModel
    _format_83(coords[1]), _format_83(coords[2]), symbol)
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-113660286.82573535" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-103312265.56089666" could not be represented in a width-8 field
./run_md.py:723: UserWarning: No output for .//run_0. MD run failed with The periodic box size has decreased to less than twice the nonbonded cutoff..
  warnings.warn(f"No output for {output_dir_replicate}. MD run failed with {result}.")
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 389, in writeModel
    _format_83(coords[1]), _format_83(coords[2]), symbol)
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "1239164367.0057662" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "158647840.8805343" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
./run_md.py:723: UserWarning: No output for .//run_0. MD run failed with The periodic box size has decreased to less than twice the nonbonded cutoff..
  warnings.warn(f"No output for {output_dir_replicate}. MD run failed with {result}.")
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "562458357.2523514" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-93943187.44770777" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "151502884.95437747" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "577112313.2202013" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-288363222.1261118" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-45135408.27106731" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "707566384.0744041" could not be represented in a width-8 field

1549860_parsley_1.3.0.xml
2220694_parsley_1.0.0.xml
4501702_sage_2.0.0.xml
2006075_parsley_1.0.0.xml
2002966_sage_2.0.0.xml
2225118_parsley_1.2.0.xml
2100349_parsley_1.3.0.xml
2004882_sage_2.0.0.xml
7059603_sage_2.0.0.xml
2225118_parsley_1.3.0.xml
4501702_parsley_1.3.0.xml
2218498_parsley_1.1.0.xml
2107938_parsley_1.1.0.xml
2006784_parsley_1.2.0.xml
2208497_parsley_1.1.0.xml
2012267_parsley_1.2.0.xml
1513282_parsley_1.2.0.xml
2235036_parsley_1.3.0.xml
2220312_parsley_1.3.0.xml
2239864_parsley_1.3.0.xml
4501703_sage_2.0.0.xml
2311084_parsley_1.0.0.xml
2205115_parsley_1.0.0.xml
7241270_parsley_1.0.0.xml
2208405_parsley_1.3.0.xml
2208497_parsley_1.0.0.xml
7117669_parsley_1.1.0.xml
1548052_sage_2.0.0.xml
2006784_sage_2.0.0.xml
2237654_parsley_1.0.0.xml
1549860_parsley_1.1.0.xml
7232896_sage_2.0.0.xml
1555524_parsley_1.3.0.xml
4508226_parsley_1.1.0.xml
2022026_parsley_1.3.0.xml
2108038_sage_2.0.0.xml
4506226_parsley_1.2.0.xml
2238479_parsley_1.3.0.xml
2237525_parsley_1.3.0.xml
2237654_parsley_1.2.0.xml
2200184_parsley_1.3.0.xml
7151750_parsley_1.2.0.xml
2105585_parsley_1.2.0.xml
7054714_parsley_1.2.0.xml
2212364_parsley_1.3.0.xml
7059603_parsley_1.2.0.xml
2107937_parsley_1.1.0.xml
2207637_parsley_1.2.0.xml
7210454_parsley_1.1.0.xml
7206297_parsley_1.2.0.xml
1557858_parsley_1.1.0.xml
7054714_parsley_1.1.0.xml
2240041_parsley_1.1.0.xml
2208241_sage_2.0.0.xml
7206161_parsley_1.3.0.xml
1011023_parsley_1.0.0.xml
2229006_parsley_1.1.0.xml
2022026_parsley_1.0.0.xml
2100865_parsley_1.1.0.xml
4022058_parsley_1.2.0.xml
2004431_sage_2.0.0.xml
2007155_parsley_1.1.0.xml
7219800_sage_2.0.0.xml
2217204_parsley_1.3.0.xml
2018826_parsley_1.0.0.xml
1513282_parsley_1.0.0.xml
2203632_sage_2.0.0.xml
4107711_sage_2.0.0.xml
7214681_parsley_1.0.0.xml
1555524_parsley_1.1.0.xml
1513289_parsley_1.1.0.xml
2100350_parsley_1.3.0.xml
1513592_parsley_1.3.0.xml
2219243_parsley_1.1.0.xml
7216950_parsley_1.2.0.xml
2006075_parsley_1.1.0.xml
2237770_parsley_1.3.0.xml
2231503_parsley_1.3.0.xml
8100793_parsley_1.0.0.xml
7206934_parsley_1.1.0.xml
2100662_parsley_1.3.0.xml
2208405_parsley_1.2.0.xml
2008450_sage_2.0.0.xml
1516412_parsley_1.2.0.xml
2105612_sage_2.0.0.xml
2005486_sage_2.0.0.xml
2015769_sage_2.0.0.xml
2205643_parsley_1.3.0.xml
2219974_parsley_1.1.0.xml
2014075_parsley_1.1.0.xml
2203241_parsley_1.0.0.xml
8100793_sage_2.0.0.xml
2200608_parsley_1.1.0.xml
7221763_parsley_1.2.0.xml
2004882_parsley_1.2.0.xml
1519794_sage_2.0.0.xml
1516412_sage_2.0.0.xml
2104176_parsley_1.0.0.xml
1504385_parsley_1.0.0.xml
2208843_parsley_1.0.0.xml
4339655_parsley_1.1.0.xml
7238221_parsley_1.0.0.xml
7151750_parsley_1.3.0.xml
4513593_parsley_1.1.0.xml
2217222_parsley_1.3.0.xml
4114396_parsley_1.0.0.xml
4038405_parsley_1.0.0.xml
2101393_parsley_1.0.0.xml
2229006_parsley_1.2.0.xml
7213183_parsley_1.1.0.xml
2012267_parsley_1.3.0.xml
4500369_parsley_1.2.0.xml
2243066_parsley_1.1.0.xml
2211928_parsley_1.1.0.xml
2237525_parsley_1.2.0.xml
1518526_parsley_1.3.0.xml
2103490_parsley_1.3.0.xml
7238219_parsley_1.1.0.xml
2202092_parsley_1.0.0.xml
2008677_sage_2.0.0.xml
7100390_parsley_1.0.0.xml
7214681_parsley_1.3.0.xml
2015769_parsley_1.1.0.xml
2200608_parsley_1.2.0.xml
2205115_parsley_1.1.0.xml
2012828_parsley_1.0.0.xml
4038405_parsley_1.3.0.xml
1011023_sage_2.0.0.xml
7241270_parsley_1.3.0.xml
7213183_sage_2.0.0.xml
7210454_parsley_1.2.0.xml
7059603_parsley_1.1.0.xml
7214681_parsley_1.1.0.xml
7045653_parsley_1.0.0.xml
2300006_parsley_1.2.0.xml
1557858_sage_2.0.0.xml
1555524_sage_2.0.0.xml
2107938_parsley_1.0.0.xml
4025034_parsley_1.2.0.xml
7242545_parsley_1.1.0.xml
2205643_parsley_1.0.0.xml
2007371_parsley_1.3.0.xml
2107937_parsley_1.3.0.xml
2214563_sage_2.0.0.xml
2007155_parsley_1.2.0.xml
2100147_parsley_1.3.0.xml
4035731_parsley_1.1.0.xml
2206542_parsley_1.0.0.xml
1557858_parsley_1.0.0.xml
2203632_parsley_1.3.0.xml
7236728_parsley_1.0.0.xml
2100350_sage_2.0.0.xml
7214418_parsley_1.0.0.xml
1519794_parsley_1.0.0.xml
2220694_parsley_1.3.0.xml
7236728_parsley_1.3.0.xml
4507232_parsley_1.2.0.xml
7052009_parsley_1.2.0.xml
2022073_sage_2.0.0.xml
2201333_parsley_1.0.0.xml
4022058_parsley_1.3.0.xml
2200633_parsley_1.3.0.xml
2229458_sage_2.0.0.xml
4037896_parsley_1.2.0.xml
2221170_parsley_1.0.0.xml
7200790_sage_2.0.0.xml
7232896_parsley_1.1.0.xml
2012267_parsley_1.0.0.xml
2206542_sage_2.0.0.xml
2207637_sage_2.0.0.xml
4513593_parsley_1.0.0.xml
2210911_sage_2.0.0.xml
2012267_parsley_1.1.0.xml
7216950_parsley_1.0.0.xml
2234037_parsley_1.3.0.xml
2002966_parsley_1.2.0.xml
2100348_parsley_1.2.0.xml
2104943_parsley_1.1.0.xml
2021969_sage_2.0.0.xml
2022073_parsley_1.1.0.xml
7210454_sage_2.0.0.xml
2211928_parsley_1.2.0.xml
2002966_parsley_1.3.0.xml
2100147_parsley_1.0.0.xml
2231730_parsley_1.2.0.xml
2210911_parsley_1.0.0.xml
7221763_parsley_1.0.0.xml
7206516_parsley_1.1.0.xml
7200790_parsley_1.0.0.xml
2216977_parsley_1.0.0.xml
3500018_parsley_1.1.0.xml
7117669_parsley_1.0.0.xml
4515127_parsley_1.1.0.xml
7206297_parsley_1.3.0.xml
4516576_parsley_1.1.0.xml
1557858_parsley_1.2.0.xml
7219800_parsley_1.0.0.xml
7151750_sage_2.0.0.xml
2201333_parsley_1.1.0.xml
1518526_parsley_1.2.0.xml
2311084_parsley_1.2.0.xml
2231307_parsley_1.0.0.xml
7151616_parsley_1.1.0.xml
2100662_sage_2.0.0.xml
4116819_sage_2.0.0.xml
2021969_parsley_1.2.0.xml
7206516_parsley_1.0.0.xml
2209888_parsley_1.1.0.xml
2003630_parsley_1.2.0.xml
2100350_parsley_1.0.0.xml
4501704_parsley_1.3.0.xml
2219243_parsley_1.0.0.xml
7238221_parsley_1.2.0.xml
2235036_parsley_1.2.0.xml
2237770_parsley_1.0.0.xml
2208241_parsley_1.3.0.xml
2005486_parsley_1.1.0.xml
2100865_parsley_1.3.0.xml
4027137_parsley_1.2.0.xml
7111201_parsley_1.3.0.xml
2105585_sage_2.0.0.xml
7121745_parsley_1.3.0.xml
2203255_parsley_1.0.0.xml
2225118_parsley_1.0.0.xml
2209888_parsley_1.2.0.xml
4037896_parsley_1.1.0.xml
2006784_parsley_1.3.0.xml
2018826_parsley_1.2.0.xml
2201001_parsley_1.0.0.xml
2243066_sage_2.0.0.xml
2236004_parsley_1.0.0.xml
2102305_parsley_1.1.0.xml
1519825_parsley_1.2.0.xml
2224771_parsley_1.0.0.xml
4513593_parsley_1.3.0.xml
7213183_parsley_1.0.0.xml
2100437_parsley_1.0.0.xml
7154682_parsley_1.2.0.xml
2215272_parsley_1.1.0.xml
7236113_parsley_1.1.0.xml
2229458_parsley_1.0.0.xml
4116140_sage_2.0.0.xml
7045653_parsley_1.3.0.xml
7118053_parsley_1.0.0.xml
2212364_parsley_1.0.0.xml
7236728_parsley_1.2.0.xml
4034456_parsley_1.3.0.xml
2100350_parsley_1.2.0.xml
7219429_parsley_1.0.0.xml
1519868_parsley_1.3.0.xml
4037916_parsley_1.1.0.xml
4500369_parsley_1.0.0.xml
2002929_parsley_1.3.0.xml
2108038_parsley_1.0.0.xml
4501703_parsley_1.3.0.xml
2237654_parsley_1.1.0.xml
1548052_parsley_1.3.0.xml
2224771_parsley_1.2.0.xml
2200184_parsley_1.1.0.xml
1511801_parsley_1.0.0.xml
7052009_parsley_1.1.0.xml
2214563_parsley_1.3.0.xml
2104943_parsley_1.3.0.xml
7206934_parsley_1.0.0.xml
2211928_parsley_1.0.0.xml
2108038_parsley_1.2.0.xml
7236113_parsley_1.0.0.xml
2107938_parsley_1.3.0.xml
2100603_parsley_1.1.0.xml
2206542_parsley_1.3.0.xml
7154682_parsley_1.0.0.xml
4506226_sage_2.0.0.xml
2105702_parsley_1.3.0.xml
2021969_parsley_1.1.0.xml
7054714_parsley_1.3.0.xml
4038405_parsley_1.1.0.xml
1504385_sage_2.0.0.xml
2235036_parsley_1.1.0.xml
2004882_parsley_1.0.0.xml
7221763_parsley_1.1.0.xml
2022053_parsley_1.3.0.xml
2105702_parsley_1.0.0.xml
2214563_parsley_1.2.0.xml
2234037_parsley_1.0.0.xml
4022058_parsley_1.0.0.xml
4506226_parsley_1.1.0.xml
2226624_parsley_1.0.0.xml
7052010_parsley_1.1.0.xml
4501702_parsley_1.0.0.xml
1548052_parsley_1.2.0.xml
7121745_parsley_1.0.0.xml
2241317_parsley_1.2.0.xml
2212135_parsley_1.1.0.xml
2231503_parsley_1.1.0.xml
1556231_parsley_1.0.0.xml
2201001_parsley_1.2.0.xml
4500369_parsley_1.3.0.xml
2205115_sage_2.0.0.xml
2236470_parsley_1.1.0.xml
2006784_parsley_1.0.0.xml
2243066_parsley_1.3.0.xml
1549860_sage_2.0.0.xml
2300006_parsley_1.3.0.xml
2006075_sage_2.0.0.xml
2240041_sage_2.0.0.xml
2100865_parsley_1.2.0.xml
2208241_parsley_1.0.0.xml
1519794_parsley_1.2.0.xml
2100840_parsley_1.1.0.xml
2237525_sage_2.0.0.xml
7117669_sage_2.0.0.xml
4513593_parsley_1.2.0.xml
2103490_parsley_1.2.0.xml
2022026_sage_2.0.0.xml
2231730_parsley_1.1.0.xml
2205115_parsley_1.3.0.xml
2221170_parsley_1.2.0.xml
2022026_parsley_1.1.0.xml
1511801_parsley_1.3.0.xml
7238219_parsley_1.2.0.xml
7157439_parsley_1.0.0.xmlTraceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-106910462.47187762" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-383313771.12337536" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "142331597.0556473" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-349288760.28094125" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "146649421.00160933" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "170636549.39392745" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "396595630.5144236" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-8704582847.208462" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-281046876.59955806" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "276868944.74600285" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-49617437.45754752" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-116162059.62263498" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 389, in writeModel
    _format_83(coords[1]), _format_83(coords[2]), symbol)
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "252523791.76778203" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "141311582.20691288" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-2048121348.0660133" could not be represented in a width-8 field
./run_md.py:723: UserWarning: No output for .//run_0. MD run failed with The periodic box size has decreased to less than twice the nonbonded cutoff..
  warnings.warn(f"No output for {output_dir_replicate}. MD run failed with {result}.")
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "2296071698.5214105" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "302837358.60171205" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-534299353.533579" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "7085132416.714685" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "137366472.46429145" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "401708118.5345638" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-450956237.1626406" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-791625232.0542513" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-211635247.51289356" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "175283953.111381" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "236886614.11022955" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "752198908.8211578" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-171209396.9349405" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "236412170.93061075" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "1104314965.513593" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'

1513592_parsley_1.2.0.xml
2207637_parsley_1.1.0.xml
2238379_sage_2.0.0.xml
2017090_sage_2.0.0.xml
2202092_sage_2.0.0.xml
7219429_sage_2.0.0.xml
2105585_parsley_1.0.0.xml
2108038_parsley_1.3.0.xml
7236113_parsley_1.2.0.xml
2206832_parsley_1.3.0.xml
7121745_parsley_1.2.0.xml
2208405_parsley_1.0.0.xml
2022073_parsley_1.2.0.xml
2311654_parsley_1.1.0.xml
2006075_parsley_1.2.0.xml
2205937_parsley_1.1.0.xml
7154682_parsley_1.1.0.xml
8100793_parsley_1.1.0.xml
2003630_sage_2.0.0.xml
1548052_parsley_1.0.0.xml
2002929_parsley_1.2.0.xml
2014075_parsley_1.3.0.xml
4513593_sage_2.0.0.xml
2208241_parsley_1.1.0.xml
4114396_parsley_1.3.0.xml
1549860_parsley_1.2.0.xml
1504357_parsley_1.3.0.xml
7206161_parsley_1.2.0.xml
7236728_parsley_1.1.0.xml
7157439_parsley_1.3.0.xml
2008677_parsley_1.0.0.xml
1513289_parsley_1.2.0.xml
2238479_sage_2.0.0.xml
7214418_parsley_1.2.0.xml
2021969_parsley_1.3.0.xml
7242545_parsley_1.2.0.xml
2208241_parsley_1.2.0.xml
1519868_parsley_1.2.0.xml
2107938_parsley_1.2.0.xml
2015769_parsley_1.2.0.xml
1511816_parsley_1.1.0.xml
2203241_sage_2.0.0.xml
4116140_parsley_1.2.0.xml
2100840_parsley_1.3.0.xml
7206297_parsley_1.0.0.xml
7052010_parsley_1.2.0.xml
2104943_parsley_1.2.0.xml
2008450_parsley_1.3.0.xml
2206832_parsley_1.0.0.xml
2200633_parsley_1.0.0.xml
1519191_parsley_1.1.0.xml
4107711_parsley_1.0.0.xml
2017090_parsley_1.0.0.xml
2100437_sage_2.0.0.xml
2100348_sage_2.0.0.xml
7235707_parsley_1.0.0.xml
4501702_parsley_1.1.0.xml
2241317_parsley_1.1.0.xml
2108038_parsley_1.1.0.xml
2008677_parsley_1.3.0.xml
2200184_parsley_1.0.0.xml
7238219_parsley_1.0.0.xml
7054714_parsley_1.0.0.xml
4507232_parsley_1.3.0.xml
2101393_sage_2.0.0.xml
2217222_sage_2.0.0.xml
2002929_sage_2.0.0.xml
7151616_parsley_1.3.0.xml
4037916_parsley_1.2.0.xml
2219243_parsley_1.2.0.xml
7213183_parsley_1.2.0.xml
1556231_parsley_1.2.0.xml
7052010_sage_2.0.0.xml
2219974_parsley_1.2.0.xml
2311084_sage_2.0.0.xml
1555524_parsley_1.0.0.xml
2218498_parsley_1.3.0.xml
4025034_parsley_1.3.0.xml
7045653_sage_2.0.0.xml
2204844_parsley_1.3.0.xml
7219235_parsley_1.1.0.xml
2104176_parsley_1.2.0.xml
4116203_parsley_1.3.0.xml
4022058_parsley_1.1.0.xml
2210911_parsley_1.1.0.xml
2225118_sage_2.0.0.xml
2229458_parsley_1.2.0.xml
2217222_parsley_1.2.0.xml
4020664_sage_2.0.0.xml
4507100_sage_2.0.0.xml
7206934_parsley_1.2.0.xml
2238806_parsley_1.0.0.xml
2237770_parsley_1.1.0.xml
1519191_sage_2.0.0.xml
2015070_sage_2.0.0.xml
4515127_parsley_1.0.0.xml
7059603_parsley_1.3.0.xml
2201001_parsley_1.3.0.xml
7214418_sage_2.0.0.xml
2100147_parsley_1.1.0.xml
1504385_parsley_1.2.0.xml
4022058_sage_2.0.0.xml
7206934_sage_2.0.0.xml
2100840_parsley_1.2.0.xml
4507100_parsley_1.1.0.xml
2100662_parsley_1.2.0.xml
2015070_parsley_1.2.0.xml
4025034_sage_2.0.0.xml
2217204_sage_2.0.0.xml
2012958_parsley_1.1.0.xml
2229458_parsley_1.1.0.xml
4501703_parsley_1.0.0.xml
4037918_parsley_1.3.0.xml
2235036_parsley_1.0.0.xml
7110463_sage_2.0.0.xml
2216977_sage_2.0.0.xml
4116819_parsley_1.2.0.xml
2212364_parsley_1.2.0.xml
1516412_parsley_1.0.0.xml
3500018_parsley_1.2.0.xml
2219243_parsley_1.3.0.xml
1513289_parsley_1.3.0.xml
2231307_parsley_1.1.0.xml
7151616_sage_2.0.0.xml
2241317_parsley_1.3.0.xml
2100603_parsley_1.2.0.xml
2211928_parsley_1.3.0.xml
2012267_sage_2.0.0.xml
1511816_parsley_1.2.0.xml
2231730_sage_2.0.0.xml
2210848_parsley_1.1.0.xml
2100349_parsley_1.2.0.xml
7052848_sage_2.0.0.xml
2104176_parsley_1.3.0.xml
4116203_parsley_1.0.0.xml
2219974_parsley_1.3.0.xml
2236470_parsley_1.3.0.xml
2212135_sage_2.0.0.xml
2104943_sage_2.0.0.xml
2004431_parsley_1.2.0.xml
7111201_sage_2.0.0.xml
2240041_parsley_1.0.0.xml
2237770_sage_2.0.0.xml
2225118_parsley_1.1.0.xml
2002966_parsley_1.1.0.xml
1504357_parsley_1.0.0.xml
1513282_parsley_1.1.0.xml
2212364_parsley_1.1.0.xml
2214563_parsley_1.1.0.xml
2017090_parsley_1.2.0.xml
2220694_parsley_1.1.0.xml
7117669_parsley_1.2.0.xml
2007371_parsley_1.0.0.xml
2238379_parsley_1.0.0.xml
1011023_parsley_1.2.0.xml
7157439_parsley_1.1.0.xml
7238219_parsley_1.3.0.xml
2102305_parsley_1.3.0.xml
2216200_parsley_1.2.0.xml
2209888_parsley_1.3.0.xml
2204844_parsley_1.2.0.xml
4027137_sage_2.0.0.xml
1504357_parsley_1.2.0.xml
4506226_parsley_1.3.0.xml
2105585_parsley_1.1.0.xml
2216200_parsley_1.3.0.xml
2217204_parsley_1.2.0.xml
2212078_parsley_1.0.0.xml
2100147_sage_2.0.0.xml
7052010_parsley_1.0.0.xml
2100865_sage_2.0.0.xml
1519868_parsley_1.0.0.xml
2003630_parsley_1.3.0.xml
7242545_parsley_1.3.0.xml
2102305_parsley_1.0.0.xml
2226624_sage_2.0.0.xml
7219235_parsley_1.3.0.xml
7235707_sage_2.0.0.xml
4501703_parsley_1.1.0.xml
2014075_sage_2.0.0.xml
2217204_parsley_1.0.0.xml
1010060_parsley_1.1.0.xml
7157439_sage_2.0.0.xml
1519825_parsley_1.3.0.xml
7210454_parsley_1.0.0.xml
2203540_parsley_1.1.0.xml
2216977_parsley_1.1.0.xml
2204844_parsley_1.0.0.xml
2226624_parsley_1.3.0.xml
2231307_sage_2.0.0.xml
2104176_parsley_1.1.0.xml
7100390_parsley_1.3.0.xml
2311084_parsley_1.1.0.xml
7052010_parsley_1.3.0.xml
7151750_parsley_1.1.0.xml
4035731_parsley_1.0.0.xml
2204844_parsley_1.1.0.xml
2015769_parsley_1.0.0.xml
2107937_sage_2.0.0.xml
4507100_parsley_1.2.0.xml
2231503_sage_2.0.0.xml
1513592_parsley_1.0.0.xml
2206832_parsley_1.1.0.xml
2216200_sage_2.0.0.xml
2005486_parsley_1.3.0.xml
1557858_parsley_1.3.0.xml
2007511_parsley_1.2.0.xml
2300006_parsley_1.1.0.xml
2007155_parsley_1.3.0.xml
4116203_sage_2.0.0.xml
2100348_parsley_1.3.0.xml
7228868_parsley_1.1.0.xml
7228868_parsley_1.2.0.xml
7206516_sage_2.0.0.xml
2229006_sage_2.0.0.xml
1010060_sage_2.0.0.xml
1518526_parsley_1.0.0.xml
2217222_parsley_1.1.0.xml
7118053_parsley_1.1.0.xml
7052848_parsley_1.2.0.xml
4339655_parsley_1.0.0.xml
2018826_parsley_1.3.0.xml
4116819_parsley_1.1.0.xml
2231307_parsley_1.2.0.xml
2102305_parsley_1.2.0.xml
7219800_parsley_1.3.0.xml
4501702_parsley_1.2.0.xml
2201333_sage_2.0.0.xml
4037918_parsley_1.0.0.xml
2215272_parsley_1.2.0.xml
2243066_parsley_1.2.0.xml
1100989_parsley_1.3.0.xml
2210848_parsley_1.2.0.xml
2204844_sage_2.0.0.xml
7117669_parsley_1.3.0.xml
2226624_parsley_1.1.0.xml
7228868_parsley_1.0.0.xml
2012828_parsley_1.1.0.xml
2223130_parsley_1.1.0.xml
7206161_parsley_1.1.0.xml
2208843_parsley_1.1.0.xml
2100437_parsley_1.3.0.xml
2223130_parsley_1.3.0.xml
2100603_sage_2.0.0.xml
7121743_sage_2.0.0.xml
4107711_parsley_1.1.0.xml
2208497_sage_2.0.0.xml
7200790_parsley_1.1.0.xml
7238221_parsley_1.1.0.xml
2021969_parsley_1.0.0.xml
1519794_parsley_1.1.0.xml
1011023_parsley_1.3.0.xml
2012958_parsley_1.2.0.xml
2210848_parsley_1.0.0.xml
2216200_parsley_1.0.0.xml
7154682_parsley_1.3.0.xml
2300006_parsley_1.0.0.xml
7232896_parsley_1.0.0.xml
7221763_parsley_1.3.0.xml
2200184_sage_2.0.0.xml
7220218_parsley_1.1.0.xml
7214418_parsley_1.1.0.xml
2212364_sage_2.0.0.xml
1100992_parsley_1.2.0.xml
7216950_sage_2.0.0.xml
2205937_parsley_1.0.0.xml
2200184_parsley_1.2.0.xml
2006099_sage_2.0.0.xml
2217204_parsley_1.1.0.xml
2221170_parsley_1.1.0.xml
4507100_parsley_1.0.0.xml
2203241_parsley_1.1.0.xml
4116203_parsley_1.2.0.xml
7214681_parsley_1.2.0.xml
2015070_parsley_1.0.0.xml
2201333_parsley_1.2.0.xml
2201333_parsley_1.3.0.xml
2205937_parsley_1.3.0.xml
7238218_parsley_1.2.0.xml
2107937_parsley_1.2.0.xml
2006099_parsley_1.3.0.xml
2014075_parsley_1.2.0.xml
2216977_parsley_1.2.0.xml
7228868_parsley_1.3.0.xml
7206516_parsley_1.2.0.xml
7200790_parsley_1.3.0.xml
2239864_parsley_1.1.0.xml
7054714_sage_2.0.0.xml
2006784_parsley_1.1.0.xml
2207637_parsley_1.0.0.xml
2239864_parsley_1.0.0.xml
2100437_parsley_1.1.0.xml
2006099_parsley_1.0.0.xml
4508226_parsley_1.3.0.xml
2008450_parsley_1.1.0.xml
4500369_sage_2.0.0.xml
2202092_parsley_1.2.0.xml
2224771_parsley_1.1.0.xml
2236004_parsley_1.2.0.xml
4037916_parsley_1.0.0.xml
2022053_parsley_1.2.0.xml
2210848_sage_2.0.0.xml
2237654_sage_2.0.0.xml
1504385_parsley_1.1.0.xml
4516576_sage_2.0.0.xml
7232896_parsley_1.3.0.xml
4034456_sage_2.0.0.xml
2236004_sage_2.0.0.xml
4500369_parsley_1.1.0.xml
2239864_parsley_1.2.0.xml
7238219_sage_2.0.0.xml
2236004_parsley_1.1.0.xml
4508226_parsley_1.0.0.xml
4037918_parsley_1.1.0.xml
1556231_parsley_1.1.0.xml
4020664_parsley_1.0.0.xml
1519868_parsley_1.1.0.xml
2002929_parsley_1.0.0.xml
2208405_parsley_1.1.0.xml
2311654_parsley_1.2.0.xml
2234037_parsley_1.2.0.xml
1555524_parsley_1.2.0.xml
2208497_parsley_1.3.0.xmlTraceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-31277590.814292487" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-2592014006.148057" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-254662434.02478546" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-227754001.13150463" could not be represented in a width-8 field
usage: run_md.py xml [-h] [--nvt] --input INPUT --pdb PDB [--prefix PREFIX]
                     [--nanoseconds NANOSECONDS] [--replicates REPLICATES]
                     [--temperature TEMPERATURE]
run_md.py xml: error: argument --temperature/-t: invalid float value: 'None'
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "2165723126.642379" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 389, in writeModel
    _format_83(coords[1]), _format_83(coords[2]), symbol)
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-281680487.4666879" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "397099662.3010789" could not be represented in a width-8 field
./run_md.py:676: UserWarning: ./MIN/3500018_sage_2.0.0_min.pdb not found. Skipping.
  warnings.warn(f"{input_dict[output_dir]['pdb']} not found. Skipping.")
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "-57032504.15511869" could not be represented in a width-8 field
Traceback (most recent call last):
  File "./run_md.py", line 733, in <module>
    entry_point()
  File "./run_md.py", line 728, in entry_point
    main()
  File "./run_md.py", line 707, in main
    prefix = prefix,
  File "./run_md.py", line 359, in run_xtal_md
    fopen
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 388, in writeModel
    recordName, atomIndex%100000, atomName, resName, chainName, resId, resIC, _format_83(coords[0]),
  File "/home/yu-tang/anaconda3/envs/xtalmd/lib/python3.7/site-packages/openmm/app/pdbfile.py", line 472, in _format_83
    'in a width-8 field' % f)
ValueError: coordinate "546279903.88323" could not be represented in a width-8 field

7206161_parsley_1.0.0.xml
2311084_parsley_1.3.0.xml
2237525_parsley_1.1.0.xml
2012828_parsley_1.2.0.xml
2100349_sage_2.0.0.xml
2201001_parsley_1.1.0.xml
7219800_parsley_1.1.0.xml
2005486_parsley_1.2.0.xml
2022026_parsley_1.2.0.xml
2200633_parsley_1.1.0.xml
2242758_sage_2.0.0.xml
2203540_parsley_1.0.0.xml
7220218_parsley_1.2.0.xml
2237654_parsley_1.3.0.xml
2007155_parsley_1.0.0.xml
7220218_parsley_1.3.0.xml
2007371_parsley_1.1.0.xml
2105585_parsley_1.3.0.xml
2100348_parsley_1.0.0.xml
1518526_sage_2.0.0.xml
2238479_parsley_1.0.0.xml
4027137_parsley_1.3.0.xml
2006099_parsley_1.2.0.xml
7228868_sage_2.0.0.xml
2239864_sage_2.0.0.xml
7151750_parsley_1.0.0.xml
2211928_sage_2.0.0.xml
4339655_parsley_1.2.0.xml
1519825_parsley_1.1.0.xml
2237770_parsley_1.2.0.xml
1548052_parsley_1.1.0.xml
7052009_parsley_1.3.0.xml
7157439_parsley_1.2.0.xml
2205643_parsley_1.2.0.xml
2012958_parsley_1.3.0.xml
7052848_parsley_1.3.0.xml
4037918_sage_2.0.0.xml
2102305_sage_2.0.0.xml
1519794_parsley_1.3.0.xml
2212078_parsley_1.1.0.xml
4339655_sage_2.0.0.xml
2104176_sage_2.0.0.xml
2311654_parsley_1.0.0.xml
7110463_parsley_1.3.0.xml
4516576_parsley_1.2.0.xml
4020664_parsley_1.1.0.xml
2101393_parsley_1.3.0.xml
2212078_sage_2.0.0.xml
2231503_parsley_1.2.0.xml
4339655_parsley_1.3.0.xml
2216977_parsley_1.3.0.xml
4038405_sage_2.0.0.xml
2200633_sage_2.0.0.xml
1100989_sage_2.0.0.xml
2214563_parsley_1.0.0.xml
2004431_parsley_1.0.0.xml
1519825_sage_2.0.0.xml
2203241_parsley_1.3.0.xml
7214418_parsley_1.3.0.xml
2236470_sage_2.0.0.xml
3500018_sage_2.0.0.xml
2243066_parsley_1.0.0.xml
4507232_parsley_1.0.0.xml
2205115_parsley_1.2.0.xml
2100603_parsley_1.0.0.xml
2212135_parsley_1.2.0.xml
7216950_parsley_1.1.0.xml
2100349_parsley_1.1.0.xml
7235707_parsley_1.1.0.xml
1513282_parsley_1.3.0.xml
1504357_sage_2.0.0.xml
2218498_sage_2.0.0.xml
Run 1040 md parsley/sage files success
